src/store/actionTypes.js

```diff
@@ -3,10 +3,13 @@ export const START_RETURN__REQUEST = "START_RETURN:REQUEST"
 export const START_RETURN__SUCCESS = "START_RETURN:SUCCESS"
 export const START_RETURN__FAILURE = "START_RETURN:FAILURE"
 
+export const DROPDOWN = "DROPDOWN"
```

Comment:
You can include a dropdown action, but the redux actions are generally for higher-level things that apply to the operations the program can perform, rather than the specific mechanics.

The smaller details (like clicking on a dropdown and selecting a value) can usually be left to be handled within that component.

In this specific case it would probably be something like:

* Dropdown has props to populate the items
* When an item is selected, dropdown runs a function property like `onItemSelected` or whatever you want to call it
* The outer component (e.g., an order item) receives that, and updates the state of the order item component
* The order item component runs a function property like `onChanged` to let the outer component know that the order item has been updated
* The outer component (e.g., the order component) receives that message and updates its data
* The order component *sends an action to the redux store*